File: 백준/Bronze/1076. 저항/저항.py
c = ''
for i in range(2):
    s = input()
    if s == 'black':
        c += '0'
    elif s == 'brown':
        c += '1'
    elif s == 'red':
        c += '2'
    elif s == 'orange':
        c += '3'
    elif s == 'yellow':
        c += '4'
    elif s == 'green':
        c += '5'
    elif s == 'blue':
        c += '6'
    elif s == 'violet':
        c += '7'
    elif s == 'grey':
        c += '8'
    else :
        c += '9'
c = int(c)
s = input()
if s == 'black':
    print(c)
elif s == 'brown':
   print(c*10)
elif s == 'red':
    print(c*100)
elif s == 'orange':
    print(c*1000)
elif s == 'yellow':
    print(c*10000)
elif s == 'green':
    print(c*100000)
elif s == 'blue':
    print(c*1000000)
elif s == 'violet':
    print(c*10000000)
elif s == 'grey':
    print(c*100000000)
else :
    print(c*1000000000)

File: 백준/Bronze/1076. 저항/저항.java
import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] s = {"black","brown","red","orange","yellow","green","blue","violet","grey","white"};
        int[] arr = {1,10,100,1000,10000,100000,1000000,10000000,100000000,1000000000};
        String a = sc.nextLine();
        String b = sc.nextLine();
        String c = sc.nextLine();
        String ch = "";
        long x = 1;
        for (int i = 0; i < 10; i++) {
            if (a.equals(s[i])) ch += i;
        }
        for (int i = 0; i < 10; i++) {
            if (b.equals(s[i])) ch += i;
        }
        for (int i = 0; i < 10; i++) {
            if (c.equals(s[i])) {
                x = Integer.parseInt(ch);
                x *= arr[i];
            }
        }
        System.out.println(x);
    }
}